maintenance_plan/tests/test_maintenance_plan.py

```diff
@@ -30,6 +35,20 @@ def setUp(self):
         today = fields.Date.today()
         self.today_date = fields.Date.from_string(today)
 
+    def test_name_get(self):
+        self.assertEqual(
```

Comment:
Can you make the test unrelated to demo data. It is usually a bad plan to make it related to demo data.

Reply:
Done!